src/lib/exportTripsToCSV.spec.js

```diff
@@ -0,0 +1,51 @@
+import { exportTripsToCSV } from 'src/lib/exportTripsToCSV'
+import { getOrCreateAppFolderWithReference } from 'src/lib/getOrCreateAppFolderWithReference'
+import { mockTimeserie, mockSerie } from 'test/mockTrip'
+
+jest.mock('src/lib/getOrCreateAppFolderWithReference', () => ({
+  getOrCreateAppFolderWithReference: jest.fn()
+}))
+jest.mock('cozy-client', () => ({
+  ...jest.requireActual('cozy-client'),
+  query: jest.fn(),
+  models: {
+    file: {
+      uploadFileWithConflictStrategy: jest.fn(() => ({
+        data: { _id: 'fileId00', name: 'fileName00' }
+      }))
+    }
+  }
+}))
+
+describe('exportTripsToCSV', () => {
+  const t = jest.fn()
+  const mockClient = (mockData = []) => {
+    const client = {
+      query: jest.fn(() => ({
+        data: [mockData]
+      }))
+    }
+    return client
+  }
+  getOrCreateAppFolderWithReference.mockReturnValue({
+    _id: 'folderId00',
+    path: '/Path/To/Folder'
+  })
+
+  it('should return correctly formatted trips for the CSV file', async () => {
```

Comment:
nit: add tests (100 lines for 1 test is not enough) on this file or add a todo later issue.